introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+### Node Types
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes
+are not aware of cluster-wide distribution policies, and otherwise look just as if they were
+single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
+from accidentally executing operations on data nodes that might endanger the data consistency
+of distributed hypertables.
+
+It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
+
+### Distributed Databases
+
+Distributed databases represent the highest level in the hierarchy used to define a multi-node
+TimescaleDB Cluster. Users can define access nodes and data nodes at the database level. This means
+that if you have multiple databases in a TimescaleDB Cluster, you could theoretically have
+different nodes play both the role of access node and data node for different databases.
+We do not typically suggest running in this manner. Instead, we recommend that you
+only have one access node that acts as a query routing layer between your application and
+all of your backend data nodes.
+
+### Distributed Hypertables
+
+As discussed above, distributed databases contain distributed hypertables. Distributed hypertables are essentially
+the same as regular hypertables, except that they have an additional layer of abstraction where
+access nodes must be able to logically manage distributed hypertables. Distributed hypertables
+are also composed of chunks.
+
+With distributed hypertables, we suggest partitioning the hypertables both by space and by time.
+If you only partition data by time, that chunk will have to fill up before the access node chooses another
+data node to store the next chunk. On the other hand, if you specify a space partition, the access node
```

Comment:
"that chunk will have to fill up before the access node chooses another data node to store the next chunk, and thus during that chunk's time interval, all writes to the latest interval will be handled by a single data node, rather than load balanced across all available data nodes."

Reply:
Done